core/src/subgraph/runner.rs

```diff
@@ -475,7 +474,14 @@ where
 
         for info in created_data_sources {
             // Try to instantiate a data source from the template
-            let data_source = DataSource::try_from(info)?;
+            let data_source = match DataSource::from_template_info(info) {
+                Ok(ds) => ds,
+                Err(e @ DataSourceCreationError::Ignore(..)) => {
```

Comment:
This prolly more of a nitpick but I think I would prefer a Ok(None) over an error called Ignore as you'd not know there's an ignore from looking at the signature

Reply:
I started with that but `None` does not carry the reason for ignoring, which is helpful for debugging.